x-pack/plugins/maps/public/routing/maps_router.tsx

```diff
@@ -42,11 +47,19 @@ export async function renderApp(context, { appBasePath, element, history, onAppL
   };
 }
 
-const App = ({ history, appBasePath, onAppLeave }) => {
+interface AppProps {
```

Comment:
Mind just naming `Props`? We have been using this convection when naming interfaces fro react properties to keep things simple and consistent across components.

Reply:
Fixed this.